app/pages/5_Model_assumptions.py

```diff
@@ -0,0 +1,83 @@
+
+import streamlit as st
+import pandas as pd
+import geopandas as gpd
+import plotly.subplots as sp
+import plotly.graph_objects as go
+
+st.title("Modelling assumptions")
+
+st.header("Assumptions for technology costs")
+
+st.markdown(
+    "Model assumptions are under constant revision."
+    "Assumptions used for this study are basted on the "
+    "[technology-data repository v0.6.2](https://github.com/PyPSA/technology-data/blob/master/outputs/costs_2030.csv). "
+    "The most relevant technologies are outlined below:"
+    ""
+)
+
+url = "https://raw.githubusercontent.com/PyPSA/technology-data/master/outputs/costs_2030.csv"
+costs = pd.read_csv(url, index_col=0)
+
+considered = [
+    'central air-sourced heat pump', 'decentral air-sourced heat pump',
+    'central ground-sourced heat pump', 'decentral ground-sourced heat pump',
+    'central resistive heater', 'decentral resistive heater',
+    'central solar thermal', 'decentral solar thermal',
+    'central solid biomass CHP',
+    'central water tank storage', 'decentral water tank storage',
+    'water tank charger', 'water tank discharger',
+
+    'battery inverter', 'battery storage',
+    'biogas upgrading', 'direct air capture', 'electricity distribution grid', 'electricity grid connection',
+    'electrolysis', 'fuel cell', 'central gas CHP', 'central gas boiler',
+    'decentral gas boiler', 'hydrogen storage underground', 'hydrogen storage tank type 1',
+    'HVDC inverter pair', 'helmeth', 'micro CHP', 'OCGT', 'offwind',
+    'onwind', 'PHS', 'ror', 'methanation', 'solar-rooftop',
+    'solar-utility', 'Steam methane reforming'
+]
+
+st.write(costs.loc[considered])
```

Comment:
We can make the table nicer if using `st.table` functionality. It could be also worth to align formatting of the table with one from the Scenario tab to keep consistent the outline of the app.

Formatting is defined there: https://github.com/ekatef/heat-peaks-lit/blob/8231892645066dd84a6a2bcfe49d439ec2bfcae1/app/pages/1_Scenario_comparison.py#L111-L134